client/components/Nav.jsx

```diff
@@ -239,18 +239,18 @@ class Nav extends React.PureComponent {
               onClick={this.toggleDropdown.bind(this, 'edit')}
               onBlur={this.handleBlur}
               onFocus={this.clearHideTimeout}
+              onMouseOver={() => {
+                if (this.state.dropdownOpen !== 'none') {
+                  this.setDropdown('edit');
+                }
+              }}
             >
               <span className="nav__item-header">Edit</span>
               <InlineSVG className="nav__item-header-triangle" src={triangleUrl} />
             </button>
-            <ul className="nav__dropdown">
-              <button
-                onClick={this.toggleDropdown.bind(this, 'edit')}
-                className="nav__dropdown-heading"
-              >
-                <span>Edit</span>
-                <InlineSVG src={triangleUrl} />
-              </button>
+            <ul
+              className="nav__dropdown"
+            >
```

Comment:
Move everything in one line.

Reply:
i agree—this doesn't need to be on three lines.